arthur-maven-plugin/src/main/java/org/apache/geronimo/arthur/maven/mojo/ArthurMojo.java

```diff
@@ -194,8 +195,9 @@ private String buildDownloadUrl(final String graalPlatform) {
         final String graalJavaVersion = versionIncludesJavaVersion ?
                 versionSegments[versionSegments.length - 1].substring(1) :
                 System.getProperty("java.version", "1.8").startsWith("8") ? "8" : "11";
-        final String githubPlatform = graalPlatform.contains("win") ?
-                "windows-amd64" : (graalPlatform.contains("linux") ? "linux-amd64" : "darwin-amd64");
+        final String githubPlatform = graalPlatform.toLowerCase(ROOT).contains("win") ?
+                "windows-amd64" : (graalPlatform.toLowerCase(ROOT).contains("linux") ?
```

Comment:
is it an option to reverse the test to fallback on linux (as before) and not on macos?
rest looks perfect